workers/unity/Assets/Playground/Scripts/Bootstrap.cs

```diff
@@ -68,19 +68,14 @@ public void Awake()
                     ? WorkerRegistry.CreateWorker<UnityClient>($"{workerType}-{Guid.NewGuid()}", new Vector3(0, 0, 0))
                     : WorkerRegistry.CreateWorker(workerType, workerId, new Vector3(0, 0, 0));
 
-                if (worker == null)
```

Comment:
why did this get removed? The worker could still be null

Reply:
I dont think it can be null anymore because now an exception is thrown in the case where it was null.